docker/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM python:3.6-alpine
```

Comment:
Alpine Linux definitely results in smaller docker images. However, since our production setup is Ubuntu, it's possible that Ubuntu is more appropriate. I don't have a very strong opinion about it though.

Reply:
I agree. Ubuntu is probably better.